pi-manage

```diff
@@ -579,6 +580,17 @@ def createdb(stamp=False):
         migration_dir = os.path.dirname(os.path.abspath(p[0]))
         fm_stamp(directory=migration_dir)
     db.session.commit()
+    warnings.warn("The function createdb() will be removed in Version 4.0", DeprecationWarning)
```

Comment:
I would put it the other way round.

I would put the code into "create_tables" and in "createdb" simpley call `create_tables()`.

This way we do not need to move code in version 4.0.